docs/contributing/doing.md

```diff
@@ -0,0 +1,141 @@
+---
+id: doing
+title: Contributing
+sidebar_label: Working
+---
+
+# Working
```

Comment:
Is this working or is this doing? because you have an id and a filename that are "doing" yet the title and label are "working"

Reply:
Depends on IA. This is what i chose for now.